binaries/go-http-daemon/main.go

```diff
@@ -83,14 +83,14 @@ func executeRequest(c echo.Context) error {
 		req = daemonRequest.ToRequest()
 	}
 
-	requestResponsePair, responseErr := request.ExecuteRequest(req, daemonRequest.Profiles, daemonRequest.Variables)
+	requestResponsePairs, responseErr := request.ExecuteRequest(req, daemonRequest.Profiles, daemonRequest.Variables)
```

Comment:
why is this called `requestResponsePairs` / why was it called `requestResponsePair` beforehand? Why not `requestResponses` since it's `[]request.ExecutedRequestResponse`? I guess, why the `Pair`?

Reply:
:thinking: good point. I'll rename it.